### PR TITLE
Feature lwdev 9159 migrate to new logging

### PR DESCRIPTION
**Pull request by dmitrysmall**
Please, see: https://lykkex.atlassian.net/browse/LWDEV-9159.

### CODE REVIEW

File: src/Lykke.Service.Stellar.Api/Startup.cs [hunk 1]
```diff
@@ -62,21 +65,47 @@ public IServiceProvider ConfigureServices(IServiceCollection services)
...
 
                 var builder = new ContainerBuilder();
-                var appSettings = Configuration.LoadSettings<AppSettings>();
-
-                Log = CreateLogWithSlack(services, appSettings);
+                var appSettings = Configuration.LoadSettings<AppSettings>(options =>
+                {
+                    options.SetConnString(x => x.SlackNotifications.AzureQueue.ConnectionString);
+                    options.SetQueueName(x => x.SlackNotifications.AzureQueue.QueueName);
+                    options.SenderName = "ServiceStellarApi";
```
Comment (KonstantinRyazantsev) [inline src/Lykke.Service.Stellar.Api/Startup.cs]: Use AppEnvironment.Name

File: src/Lykke.Service.Stellar.Api/Startup.cs [hunk 2]
```diff
@@ -105,139 +134,66 @@ public void Configure(IApplicationBuilder app, IHostingEnvironment env, IApplica
...
             catch (Exception ex)
             {
-                Log?.WriteFatalErrorAsync(nameof(Startup), nameof(Configure), "", ex).GetAwaiter().GetResult();
+                Log?.Critical(ex);
                 throw;
             }
         }
 
-        private async Task StartApplication()
+        private void StartApplication()
```
Comment (KonstantinRyazantsev) [inline src/Lykke.Service.Stellar.Api/Startup.cs]: Why this method has been made sync?

File: src/Lykke.Service.Stellar.Api/Startup.cs [hunk 3]
```diff
@@ -105,139 +134,66 @@ public void Configure(IApplicationBuilder app, IHostingEnvironment env, IApplica
...
             catch (Exception ex)
             {
-                await Log.WriteFatalErrorAsync(nameof(Startup), nameof(StartApplication), "", ex);
+                Log.Critical(ex);
                 throw;
             }
         }
 
-        private async Task StopApplication()
+        private void StopApplication()
```
Comment (KonstantinRyazantsev) [inline src/Lykke.Service.Stellar.Api/Startup.cs]: and this

File: src/Lykke.Service.Stellar.Api/Startup.cs (hunk 2 above)
Comment (dmitrysmall) [reply src/Lykke.Service.Stellar.Api/Startup.cs]: We use it synchronously anyway:
`appLifetime.ApplicationStarted.Register(() => StartApplication().GetAwaiter().GetResult());`
so, I considered it to be a bit less messy if the method was synchronous by itself using `GetAwaiter().GetResult()` inside:
`appLifetime.ApplicationStarted.Register(StartApplication);`

File: src/Lykke.Service.Stellar.Api/Startup.cs (hunk 3 above)
Comment (dmitrysmall) [reply src/Lykke.Service.Stellar.Api/Startup.cs]: The same.

File: src/Lykke.Service.Stellar.Api/Startup.cs (hunk 2 above)
Comment (KonstantinRyazantsev) [reply src/Lykke.Service.Stellar.Api/Startup.cs]: See #3

**Pull request merged**